### DISCUSSION TITLE
GraphRAG for .net and ability to use in Semantic kernal

### DISCUSSION THREAD

**Discussion by ravindranathw** [microsoft/semantic-kernel#7405] [Ideas]
Python has library to do graphRag. In .net there is nothing available. Any plans for this and to use it via sementic kernal

**Comment by sophialagerkranspandey** [comment]
The correct repo for this question is here: https://github.com/microsoft/graphrag
We'd recommend you ask your question in the repo linked above.

**Reply by houstonhaynes** [reply]
I think the original post was looking for a .NET implementation of GraphRAG. Here's a custom implementation someone is keeping public notes on using C# and Neo4J https://jasonhaley.com/2024/08/06/study-notes-graph-rag1-code-sample-notebook/
